data/presets/area/footway.json

```diff
@@ -0,0 +1,21 @@
+{
+    "fields": [
+        "surface"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "terms": [
+        "area:highway",
```

Comment:
By the way, this preset will already be searchable by “footway” regardless of language, because iD will match tag values: https://github.com/openstreetmap/iD/issues/8869#issuecomment-1004306198. If matching keys is also desired, maybe that should be a more general, non-preset-specific feature on the iD side, so other consumers like Overpass turbo can decide not to match keys if it doesn’t make sense for their use case.